packages/roc-package-web-app-react/app/server/render.js

```diff
@@ -99,13 +96,11 @@ export function reactRender(url, createRoutes, store, renderPage, staticRender =
                     };
                 }
 
-                getPrefetchedData(components, locals)
+                trigger('fetch', components, locals)
                     .then(() => {
-                        let component = <RoutingContext {...renderProps} />;
+                        let component = <RouterContext {...renderProps} />;
```

Comment:
This will be awesome! Let's leave this discussion for now in this PR, because it is should have its own scope, and has nothing to do with updating dependencies

I'll create an issue with your proposed solution, then we can take it from there, ok?